module-ballerina-openapi/Ballerina.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 org= "ballerina"
 name= "openapi"
-version= "1.9.0"
+version= "@toml.version@"
 
 [[platform.java17.dependency]]
-path = "../openapi-validator/build/libs/openapi-validator-1.9.0-SNAPSHOT.jar"
+path = "../openapi-validator/build/libs/openapi-validator-@project.version@.jar"
```

Comment:
Why this version changed like this?

Reply:
It was previously committed mistakenly I guess. I revert the commit